passing `int *' as argument 3 of `getsockname(int, sockaddr *, socklen_t *)' changes signedness

--- speech_tools/utils/EST_ServiceTable.cc.orig	Thu Nov 18 23:15:21 1999
+++ speech_tools/utils/EST_ServiceTable.cc	Sun Jun 11 17:28:29 2000
@@ -296,7 +296,7 @@
 
   // This only gets the port number
 
-  if (getsockname(socket, (struct sockaddr *)&sin, SOCKLEN_CAST &size) != 0)
+  if (getsockname(socket, (struct sockaddr *)&sin, (socklen_t *) &size) != 0)
     EST_sys_error("Can't find my address");
 
 
